lib/llm/src/model_card/create.rs

```diff
@@ -157,6 +159,22 @@ impl PromptFormatterArtifact {
             .ok())
     }
 
+    pub async fn chat_template_from_repo(repo_id: &str) -> Result<Option<Self>> {
+        // Some HF model (i.e. meta-llama/Llama-4-Maverick-17B-128E-Instruct-FP8)
+        // stores the chat template as a separate file, we want to optionally store
+        // the content if it exists and put the chat template into config as normalization
+        Ok(Self::chat_template_try_is_hf_repo(repo_id)
```

Comment:
> we want to optionally store the content if it exists and put the chat template into config as normalization

Could you re-write this part? I'm having trouble understanding it.

Reply:
Maybe I should have moved the comment to `PromptFormatter` constructor. Here is just trying to record an optional file in a HF repo if it exists.